### DISCUSSION TITLE
Rendering error

### DISCUSSION THREAD

**Discussion by newera-001** [bulletphysics/bullet3#4068] [Q&A] [answered]
When I running bullet physics examplebrowser to rendering my robot, a few hours later, and then my robot got stuck.

**Comment by rohit-kumar-j** [comment, marked as answer]
@newera-001  Any output from to the terminal?

**Reply by newera-001** [reply]
nothing shows in terminal, just static

![example](https://user-images.githubusercontent.com/55731265/144773653-8fc97518-a18b-4118-bf65-0797c6e5be8f.jpg)

**Reply by newera-001** [reply]
(the info in terminal is my algorithm output)

**Reply by rohit-kumar-j** [reply]
@newera-001  Did you check your ram, etc during the crash?

**Reply by newera-001** [reply]
yes, the ram is still enough to rendering the robot ,with leaving 5g. other parameters are still good.
